css/ContactDetails.scss

```diff
@@ -23,6 +23,7 @@
 #contact-details {
 	$grid-column-gap: 20px;
 	$grid-column-width: 350px;
+	flex: 1 1 524px;
```

Comment:
Where is this number coming from?

Reply:
That was the one, that came from server, so what applied before changing to vue-classes. No more specific idea behind, just change it if you have better number... Was more like a quick-fix to get UI back to the same usable state as before.